platform/solr/solr-password/src/main/java/ddf/platform/solr/security/SolrPasswordUpdateImpl.java

```diff
@@ -161,13 +161,16 @@ private Void updatePropertiesFile() {
       passwordSavedSuccessfully = true;
       String msg =
           String.format(
-              "\"Updated encrypted Solr password in properties file %s.", systemPropertyFilename);
+              "Updated encrypted Solr password in properties file %s.", systemPropertyFilename);
       SecurityLogger.audit(msg);
       LOGGER.info(msg);
     } catch (IOException e) {
-      LOGGER.error(
-          "Exception while writing to {}. Solr password was changed, but new password was not saved.",
-          systemPropertyFilename);
+      String msgFail =
+          String.format(
+              "Exception while writing to %s. Solr password was changed, but new password was not saved.",
```

Comment:
❓ Is there any way to recover from this? Maybe we could also give some kind of action to take.

Reply:
We could point them to the documentation. There are steps there to reset the password in Solr. But I do not know of anyone log messages where we say "go read the docs". Is there something else we say that would be useful? Maybe "the application cannot use Solr until the password is restored"? What do you think?

Reply:
Yeah I like "the application cannot use solr until the password is restored" might be helpful and then maybe from there they'd figure out they need to consult the docs.

Reply:
this and others will give discrepancies between release and master - These are good suggestions and I agree with them just putting this comment out there for future reference

Reply:
Updated message.